### PR TITLE
Add runtime attach sample to customize logging level per class

### PR DESCRIPTION
**Pull request by heyams**

### CODE REVIEW

**Review by jeanbisutti — COMMENTED** [review]
I am a bit worried about using together programmatic and automatic instrumentation for logging ([see](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/log4j/log4j-appender-2.17)). We need to discuss about the impacts.

File: opentelemetry-api/runtime-attach/CustomLoggerPerClass/src/main/java/com/example/Main.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.microsoft.applicationinsights.attach.ApplicationInsights;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Main {
+
+  public static void main(String[] args) throws InterruptedException {
+    ApplicationInsights.attach();
+
+    // TODO (heya) fix duplicate log either by disabling log4j in Java agent
```
Comment (trask) [inline opentelemetry-api/runtime-attach/CustomLoggerPerClass/src/main/java/com/example/Main.java:12]: I created [work item]( https://msazure.visualstudio.com/One/_workitems/edit/24459711) we discussed previously about changing the agent so auto-instrumented log capture can be turned off without affecting manual appender configuration


```
{
  "instrumentation": {
    "logging": {
      "level": "OFF"
    }
  }
}
```

Comment (trask) [reply opentelemetry-api/runtime-attach/CustomLoggerPerClass/src/main/java/com/example/Main.java:12]: Another option: (could disable javaagent instrumentation for logback, log4j and jul)

* otel.instrumentation.logback-appender.enabled=false
* otel.instrumentation.log4j-appender.enabled=false
* otel.instrumentation.java-util-logging.enabled=false


```
{
  "instrumentation": {
    "logging": {
      "enabled": false
    }
  }
}
```

Comment (heyams) [reply opentelemetry-api/runtime-attach/CustomLoggerPerClass/src/main/java/com/example/Main.java:12]: https://github.com/microsoft/ApplicationInsights-Java/pull/3208

**Review by trask — APPROVED** [review]

**Pull request merged**